### ISSUE TITLE
muscle terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#4228]
Preferred term label (e.g., asplenia)

1. decreased compound muscle action potential amplitude 
2. abnormal circulating myomesin level 
3. increased circulating myomesin-3 level

Synonyms (e.g., absent spleen)

1. reduced compound muscle action potential amplitude, decreased CAMP amplitude, reduced CAMP amplitude
2. aberrant circulating myomesin level, aberrant circulating Myom level  
3. elevated circulating myomesin-3 level, elevated circulating Myom3 level, increased circulating Myom3 level    

Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal action potential (MP:0005402)
2. abnormal circulating protein level (MP:0005416)
3. abnormal circulating myomesin-3 level (new term)

Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. Reduced level of the compound muscle action potential (CMAP), which is recorded following electrical stimulation of a nerve from surface electrodes overlying a muscle supplied by that nerve. (HP:0033383)
2. anomaly in the amount in the blood of any of the myomesins, the structural, striated muscle proteins found in the M-line of sarcomeres.
3. higher than normal levels of circulating myomesin-3, a structural muscle protein that may enter the circulation when striated muscle is damaged.

PMID:41502415, https://en.wikipedia.org/wiki/Myomesinhttps://en.wikipedia.org/wiki/Myomesin

If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683
